src/generate_jsdoc.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="../typings/shelljs/shelljs.d.ts"/>
+import shell = require("shelljs");
+
+function generate_doc(module: string, path: string) {
+    var moduleXml = module + ".xml"
+    var jsdoc = "jsdoc -r " + path + " -t templates/haruki -d console -q format=xml > " + moduleXml;
```

Comment:
Could we try string interpolation here? Or maybe pass an argument list to the `shelljs` module (I don't sure it provides this API but we should check).